.github/workflows/publish.yaml

```diff
@@ -29,11 +29,14 @@ jobs:
       - name: Checking out last commit in release
         if: ${{ github.event_name != 'workflow_dispatch' }}
         uses: actions/checkout@v4
+        with:
```

Comment:
I just noticed that we also need to set `fetch-depth: 0` here in order to make the timestamps work in the documentation.

Reply:
So that closes #482 ? If you have already tested it, please push to this branch and merge! 🙏🏽

Reply:
Yes, If you check the development version of the docs, you can see that the timestamps were fixed there. I just forgot to also update the publish workflow.

Reply:
Ok, I'll add it then